script/import_metadata

```diff
@@ -16,7 +16,7 @@ end
 
 
 puts "Loading environment..."
-require File.join(File.dirname(__FILE__), '../config/environment.rb')
+require_relative('../config/environment')
```

Comment:
The old one wasn't working; the new one works as long the script files remain in the same relative position. If they need to move, the path can be tweaked.

These scripts don't have a notion of rails root, so there is nothing to start a path from except their own location.